Reqnroll.Generator/UnitTestProvider/NUnit3TestGeneratorProvider.cs

```diff
@@ -50,11 +51,13 @@ public virtual void SetTestMethodIgnore(TestClassGenerationContext generationCon
 
         public virtual void SetTestClassInitializeMethod(TestClassGenerationContext generationContext)
         {
+            generationContext.TestClassInitializeMethod.Attributes |= MemberAttributes.Static;
```

Comment:
I wonder why the `MemberAttributes` type isn't marked with `[Flags]` when all the documentation clearly describes it as having bitmask behaviours. This usage seems correct.

Reply:
@Code-Grump That's the stupidity of CodeDom. We can't do anything with it...